Makefile

```diff
@@ -18,11 +18,15 @@ FL_IGNORES = .git/,.github/,tests/
 # -------------------------------------------------------------------------------------------------
 # Docker configuration
 # -------------------------------------------------------------------------------------------------
-DIR = Dockerfiles/
+DIR = Dockerfiles
 FILE = Dockerfile
-IMAGE = cytopia/ansible
+IMAGE = morganchristiansson/ansible
```

Comment:
It's currently hardcoded in my dockerfile to push to my docker hub morganchristiansson/docker-ansible

I guess let's remove the pushing and just use build cache for now